docs/docs.polserver.com/pol100/configfiles.xml

```diff
@@ -1286,110 +1286,113 @@ Region  (string regionname)
-    [ShieldCProp           (string name, string value, will set a CProp on the intrinsicn shield)]
-    [ShieldCprop           (...)]
-
-    [FireDamage]            (int/(die-string value))]
-    [ColdDamage]            (int/(die-string value))]
-    [EnergyDamage]          (int/(die-string value))]
-    [PoisonDamage]          (int/(die-string value))]
-    [PhysicalDamage]        (int/(die-string value))]
-
-    [MaxHp                 (int maximum hitpoints for intrinsic weapon {default 1}]
-
-    [MoveMode              (movemode string {default 'L'})]
-    [Privs                 (string privilege) [(more privs)]...]
-    [Settings              (string priv set 'on') [(more privs)]...]
-
-    [SpeechColor           (int speech color {default 0x3B2})]
-    [SpeechFont            (int speech color {default 3})]
-
-    [UseAdjustments        (0/1 {default 1})]
-    [RunSpeed              (int 0-250 {default dexterity})]
-    [SaveOnExit            (0/1 {default 1})]
-    [MethodScript          (string scriptname)]
-    [NoDropException       (0/1 {default 0})]
-
-    [LowerReagentCost       (int/(die-string value))]
-    [SpellDamageIncrease    (int/(die-string value))]
-    [FasterCasting          (int/(die-string value))]
-    [FasterCastRecovery     (int/(die-string value))]
-    [DefenceIncrease        (int/(die-string value))]
-    [DefenceIncreaseCap     (int/(die-string value))]
-    [LowerManaCost          (int/(die-string value))]
-    [HitChance              (int/(die-string value))]
-    [FireResistCap          (int/(die-string value))]
-    [ColdResistCap          (int/(die-string value))]
-    [EnergyResistCap        (int/(die-string value))]
-    [PhysicalResistCap      (int/(die-string value))]
-    [PoisonResistCap        (int/(die-string value))]
-    [Luck                   (int/(die-string value))]
-    [SwingSpeedIncrease     (int/(die-string value))]
-
-    [LowerReagentCostMod    (int/(die-string value))]
-    [SpellDamageIncreaseMod (int/(die-string value))]
-    [FasterCastingMod       (int/(die-string value))]
-    [FasterCastRecoveryMod  (int/(die-string value))]
-    [DefenceIncreaseMod     (int/(die-string value))]
-    [DefenceIncreaseCapMod  (int/(die-string value))]
-    [LowerManaCostMod       (int/(die-string value))]
-    [HitChanceMod           (int/(die-string value))]
-    [FireResistCapMod       (int/(die-string value))]
-    [ColdResistCapMod       (int/(die-string value))]
-    [EnergyResistCapMod     (int/(die-string value))]
-    [PhysicalResistCapMod   (int/(die-string value))]
-    [PoisonResistCapMod     (int/(die-string value))]
-    [LuckMod                (int/(die-string value))]
-    [SwingSpeedIncreaseMod  (int/(die-string value))]
-
-    [CProp                  (string name, string value, will set a CProp on the npc at creation)]
-    [Cprop                  (...)]
-}
-[NpcTemplate...]
-</structure>
+      NpcTemplate (string unique templatename)
```

Comment:
I don't think it's necessary to change the initial indentation of the  `<structure>` texts. The HTML docs are rendered in a `<pre>` tag so they will have weird spaces.